e2e/tests/transfer/authz_test.go

```diff
@@ -156,54 +152,62 @@ func (suite *AuthzTransferTestSuite) TestAuthz_MsgTransfer_Succeeds() {
 			MsgTypeUrl: transfertypes.TransferAuthorization{}.MsgTypeURL(),
 		}
 
-		resp := suite.BroadcastMessages(context.TODO(), chainA, granterWallet, &msgRevoke)
-		suite.AssertTxSuccess(resp)
+		resp := s.BroadcastMessages(context.TODO(), chainA, granterWallet, &msgRevoke)
+		s.AssertTxSuccess(resp)
 	})
 
 	t.Run("exec unauthorized MsgTransfer", func(t *testing.T) {
 		transferMsg := transfertypes.MsgTransfer{
-			SourcePort:    channelA.PortID,
-			SourceChannel: channelA.ChannelID,
+			SourcePort:    chainAChannels.PortID,
+			SourceChannel: chainAChannels.ChannelID,
 			Token:         testvalues.DefaultTransferAmount(chainADenom),
 			Sender:        granterAddress,
 			Receiver:      receiverWalletAddress,
-			TimeoutHeight: suite.GetTimeoutHeight(ctx, chainB),
+			TimeoutHeight: s.GetTimeoutHeight(ctx, chainB),
 		}
 
 		protoAny, err := codectypes.NewAnyWithValue(&transferMsg)
-		suite.Require().NoError(err)
+		s.Require().NoError(err)
 
 		msgExec := &authz.MsgExec{
 			Grantee: granteeAddress,
 			Msgs:    []*codectypes.Any{protoAny},
 		}
 
-		resp := suite.BroadcastMessages(context.TODO(), chainA, granteeWallet, msgExec)
-		suite.AssertTxFailure(resp, authz.ErrNoAuthorizationFound)
+		resp := s.BroadcastMessages(context.TODO(), chainA, granteeWallet, msgExec)
+		s.AssertTxFailure(resp, authz.ErrNoAuthorizationFound)
+	})
+
+	t.Run("stop relayer", func(t *testing.T) {
+		s.StopRelayer(ctx, relayer)
 	})
 }
 
-func (suite *AuthzTransferTestSuite) TestAuthz_InvalidTransferAuthorizations() {
-	t := suite.T()
+func (s *TransferTestSuite) TestAuthz_InvalidTransferAuthorizations() {
+	t := s.T()
 	ctx := context.TODO()
 
-	relayer, channelA := suite.SetupChainsRelayerAndChannel(ctx, suite.TransferChannelOptions())
-	chainA, chainB := suite.GetChains()
+	chainA, chainB := s.GetChains()
+	relayer := s.GetRelayerFromSuite()
+
+	channelA, err := relayer.GetChannels(ctx, s.GetRelayerExecReporter(), chainA.Config().ChainID)
+	s.Require().NoError(err)
+	chainAChannels := channelA[len(channelA)-1]
```

Comment:
this seems like a good candidate for a helper function, it seems it's now used in almost all the tests!  Maybe we can add this to the `E2ETestSuite`?

Reply:
Maybe we could actually even make a helper that returns the relayer and the channel info.